### DISCUSSION TITLE
how can i get child IP ? how do i configure IP for child?

### DISCUSSION THREAD

**Discussion by mithunGhosh13** [openthread/openthread#7682] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.** A clear and concise description of what the problem is.

**Describe the solution you'd like** A clear and concise description of what you want to happen.

**Describe alternatives you've considered** A clear and concise description of any alternative solutions or features you've considered.

**Additional context** Add any other context or screenshots about the feature request here.

**Comment by jwhui** [comment, marked as answer]
Thread devices typically configure IPv6 addresses using Stateless Address Autoconfiguration (SLAAC). However, you can manually configure IPv6 addresses using the [`ipaddr add`](https://github.com/openthread/openthread/tree/main/src/cli#ipaddr-add-ipaddr) CLI.

If the Child is a Minimal Thread Device, you can retrieve the child IPv6 addresses from the child table using the [`child table`](https://github.com/openthread/openthread/tree/main/src/cli#child-table) CLI.

If the Child is a Full Thread Device, you will need to retrieve the IPv6 address using other means. One way is via DNS-SD. You can learn more in the [Thread Border Router Codelab](url[)](https://openthread.io/codelabs/openthread-border-router#0).

Note that the CLI simply uses OpenThread APIs. If you need to call OpenThread APIs, you can use the [CLI implementation](https://github.com/openthread/openthread/tree/main/src/cli) as a reference for how to call the APIs.
